### PR TITLE
DeCYFIR IoC Integration bug fixes 

### PR DESCRIPTION
**Pull request by mohanreddy2121**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @mohanreddy2121
Thank you for your contribution!
It looks great!
Please bump the pack version in the pack_metadata.json file and add notes describing your changes to a new Release Notes file.
To do that, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

and also revert the package-lock.json file

**Comment by mohanreddy2121** [comment]
Thanks @adi88d , I will update soon

**Comment by mohanreddy2121** [comment]
@adi88d , When i use this command (demisto-sdk update-release-notes -i Packs/DeCYFIR/) nothing is updated. 
can you please let me know any other steps to do.

[2024-08-19 15:08:57,308] [INFO] Either no changes were found in DeCYFIR pack or the changes found should not be documented in the release notes file.
If relevant changes were made, please commit the changes and rerun the command.

File: Packs/DeCYFIR/Integrations/decyfiriocs/decyfiriocs.yml [hunk 1]
```diff
@@ -22,93 +22,87 @@ configuration:
...
+description: DeCYFIR API's provides External Threat Landscape Management insights.
+display: DeCYFIR Indicators & Threat Intelligence Feed
+name: DeCYFIR Feed
 script:
+  commands:
+  - deprecated: false
+    description: Gets indicators from the feed.
+    execution: false
+    name: decyfir-get-indicators
   dockerimage: demisto/python3:3.10.13.84405
```
Comment (adi88d) [inline Packs/DeCYFIR/Integrations/decyfiriocs/decyfiriocs.yml:97]: Please change to the latest docker:

dockerimage: demisto/python3:3.11.9.107902

**Comment by mohanreddy2121** [comment]
Hi @adi88d , I clear all errors but only pre commit options cannot clear it out

**Comment by adi88d** [comment]
@mohanreddy2121 We have some internal issue with the docker image creation now, I will merge this pull request when it resolved. Thanks

**Comment by mohanreddy2121** [comment]
I changed the docker image version demisto/python3:3.11.9.107902
Please check it out once let me know if any thing need to change.

**Comment by adi88d** [comment]
Thanks @mohanreddy2121 , just revert the `package-lock.json` file

**Comment by mohanreddy2121** [comment]
Hi @adi88d, Thanks for update. Is anything I need to update. My side code part fixes are done, we can ready release it prod. 

I have doubt for next PR, We want develop a new version v2 for both Threat intel and xsoar, can you please let us know how we need to do the new version v2?

**Comment by mohanreddy2121** [comment]
Sure @adi88d, will do that and thanks for sharing

**Review by adi88d — APPROVED** [review]
Nice!

**Pull request merged**